### ISSUE TITLE
Deprecated highlight.js dependency in debug-utils

### ISSUE THREAD

**Issue by frangio** [trufflesuite/truffle#3551]
Getting the following message on install of `@truffle/contract` due to a deprecated dependency `highlight.js`.

`@truffle/contract > @truffle/debug-utils > highlight.js`.

```
> highlight.js@9.18.5 postinstall
> node deprecated.js

-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

  Verion 9 of Highlight.js has reached EOL.  It will no longer
  be supported or receive security updates in the future.
  Please upgrade to version 10 or encourage your indirect
  dependencies to do so.

  For more info:

  https://github.com/highlightjs/highlight.js/issues/2877
  https://github.com/highlightjs/highlight.js/blob/master/VERSION_10_UPGRADE.md

-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
```

Note: It's a large message colored with a red background so it's very annoying. :slightly_smiling_face:

**Comment by haltman-at** [comment]
Thanks for pointing this out!  Threw up a quick PR to address this.

**Comment by gnidan** [comment]
This is now released in v5.1.55

**Comment by gjgd** [comment]
I'm still getting this error, even with with the latest version. Here is the output of `npm ls highlight.js` in my project:
```
@sidetree/ethereum@0.1.1-unstable.20 /home/gjgd/sidetree.js/packages/ledger-ethereum
└─┬ @truffle/contract@4.3.1
  └─┬ @truffle/debug-utils@5.0.4
    ├─┬ @trufflesuite/chromafi@2.2.1
    │ └── highlight.js@9.18.5
    └── highlight.js@10.4.1
```

As you can see, there are two highlight.js dependencies:
- One that was fixed in https://github.com/trufflesuite/truffle/pull/3553
- One that still uses the insecure v9.18.5, because @truffle/debug-utils depends on @trufflesuite/chromafi which depends on this insecure version.

Please reopen the issue @gnidan

**Comment by haltman-at** [comment]
Thanks for the heads up!  I updated `@trufflesuite/chromafi` and have now put up a PR to update Truffle.

**Comment by gjgd** [comment]
Awesome, thanks! 🎉

**Comment by fainashalts** [comment]
This is addressed in Truffle v5.1.57. Closing!
